src/leafdoc.js

```diff
@@ -114,6 +114,13 @@ Leafdoc.prototype.registerDocumentable = function(name, label) {
 	return this;
 };
 
+// 🍂method getTemplateEngine(): Handlebars
+// Returns handlebars template engine used to render templates.
+// You can use it for override helpers or register new.
+Leafdoc.prototype.getTemplateEngine = function() {
```

Comment:
What's a use case for this? It's not clear for me what this feature can be used for.